nengo_loihi/builder/connection.py

```diff
@@ -104,10 +139,10 @@ def build_connection(model, conn):
     eval_points = None
     solver_info = None
     neuron_type = None
+    post_slice = conn.post_slice
 
-    # Sample transform if given a distribution
-    transform = get_samples(
-        conn.transform, conn.size_out, d=conn.size_mid, rng=rng)
+    # sample transform (if using a distribution)
+    transform = sample_transform(conn, rng=rng)
```

Comment:
If we sample the transform here, then we can't sample it separately in `build_decoders` if we have a weight solver, since then we'll definitely get a different sampled transform if we use weight solvers as if we don't.

Reply:
Hmm yeah let me see if there's a nice way to clean that up.

Reply:
I just ended up explicitly passing the transform to build_decoders (which I know you had in at some point and then I took out 😜)

Reply:
Yeah, that's why I did it that way :)